CLI_TEST_CHECKLIST.md

```
@@ -0,0 +1,302 @@
+# CLI Testing Checklist
+
+# CLI Testing Checklist
+
+## Test Device Info
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)